src/shared/components/Beer/Beer.jsx

```diff
@@ -1,4 +1,5 @@
 import './styles.scss';
+import PropTypes from 'prop-types';
 const Beer = ({ food, brewed, tagline, ebc, name, abv, description, img, ibu }) => {
```

Comment:
There should be a break between imports and component

Reply:
Done!